data/json/items/armor.json

```diff
@@ -226,6 +226,17 @@
     "environmental_protection": 2,
     "flags": [ "STURDY", "BELTED", "BLOCK_WHILE_WORN", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "armguard_acidchitin",
+    "copy-from": "armguard_chitin",
+    "type": "ARMOR",
+    "name": "pair of biosilicified chitin arm guards",
+    "name_plural": "pairs of biosilicified chitin arm guards",
```

Comment:
We definitely need much shorter names for all of these biosilicified items. No need to change it right in this PR but please think about it anyway.

Reply:
Yeah, I looked around for shorter names, but couldn't find anything other than just simply "hardened", "acid-tempered", "resistant chitin", etc. As far as I know, there isn't a cleaner shorter name for biosilicification, but I do agree with you. Maybe even "bios-chitin" but then that might just create confusion unless there is a mention of it in the description.

Reply:
The "bio" part isn't needed.
"Acid chitin" could be a good common name.